src/arrayMethodSort.js

```diff
@@ -4,9 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
Defining a method on `Array.prototype` directly is generally discouraged as it can lead to conflicts with other code. Consider using a different approach, such as creating a utility function that accepts an array as an argument.